reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+          description: The price of the item before discounts have been applied.  Can only be specified if this is a custom line item, in which case it is mandatory.  To change the price of a product, use `discount` as specified below.
+        grams:
+          type: string
+          description: Weight in grams.  Can only be specified if this is a custom line item.  If not specified, defaults to `0`.
+        requires_shipping:
+          type: boolean
+          description: States whether or not the fulfillment requires shipping. Values are `true` or `false`.  Can only be specified if this is a custom line item.  Defaults to `false`.
+          enum: 
+          - true
+          - false
+        sku:
+          type: string
+          description: Can only be specified if this is a custom line item.
+        taxable:
+          type: boolean
+          description: States whether or not the product is taxable. Values are `true` or `false`.  Can only be specified if this is a custom line item.  Mandatory for custom products.
+
+  product_variant_line_item:      
+    description: product variant line item
+    allOf:
+    - $ref: '#/definitions/line_item'
+    - type: object
+      required:
+      - variant_id
+       
+      properties:
+        variant_id:
+          type: string
+          description: The `id` of the product variant corresponding to the `line_item`.  Null if custom `line_item`.
+          readOnly: false
+        product_id:
+          type: string
+          description: The `id` of the product corresponding to the line item’s product variant.
+        name:
+          type: string
+          description: The readable name of the product and variant.
+        variant_title:
+          type: string
+          description: The title of the product variant.  Default value is “Custom”.
+
+    
+  
+  shipping_rate:
+    type: object
+    required:
+    - handle
+    - title
+    - price
+    properties:
+      handle:
+        type: string
+        description: handle of the shipping rate.
+      title:
+        type: string
+        description: The shipping method name.
+      price:
+        type: string
+        description: The price of the shipping rate.
+
+
+  shipping_line:
+    type: object
+    required:
+    - title
+    - price
+    - tax_lines
+    discriminator: custom
+    properties:
+      custom:
+        description: Indicates whether or not the shipping_line is a custom_shipping_line.  When set to `true` indicates that it is a custom_shipping_line.  
+        type: boolean
+        enum: 
+        - true
+        - false
+  
+      title:
+        type: string
+        description: The title of the shipping method.
+        readOnly: false
+      price:
+        type: integer
+        format: int64
+        description: The price of this shipping method.
+        readOnly: false
+      phone:
+        type: string
+        description: The phone number, if required as specified in the `shipping_rate`.
```

Comment:
Yes exactly but its not implemented. Its required at the order level and I have to look into whether theres a ton of value to add it here, which I think there is, but would like to remove it for now or mark it not implemented